src/test/java/uk/gov/di/ipv/cri/common/library/domain/personidentity/AddressTest.java

```diff
@@ -9,24 +9,74 @@
 
 class AddressTest {
     @Test
-    void shouldGetPastAddressType() {
+    void shouldGetPreviousAddressTypeWithJustValidUntil() {
```

Comment:
Please ensure that there are unit test cases for every possible scenario. Thanks

Reply:
The test still exist it was just renamed (it used a past date), I have updated the name to shouldGetPreviousAddressTypeWhenValidUntilIsPastDate, as there are are additional tests for WhenValidUntilIsToday and WhenValidUntilIsInFuture.